evalml/automl/automl_search.py

```diff
@@ -562,14 +577,7 @@ def _should_continue(self):
         if self._interrupted:
             return False
 
-        # for add_to_rankings
```

Comment:
These lines were added for `add_to_rankings` and baseline evaluation because the engine used to call the `_should_continue_callback`. The engine doesn't do that anymore so we can delete these lines!